rfc-001-announcement-channel-protocol.md

```diff
@@ -0,0 +1,28 @@
+# What content is applicable for the announcements channel
+
+## Summary
+
+What types of messages are relevant to the newly renamed announcements channel?
+
+## Problem
+
+Off the back of the Deep Work retro hosted by Chris, we had common consensus that #general too noisy.
+
+To get round that we've created a new channel, #common-room for any chatter, and renamed general to announcements.
+
+There is a lot of ambiguity about what content is suitable for the new channel.
+
+## Proposal
+
+All content in #announcements should be a notification/message that everyone should have visibility on.
+
+In order to keep the channel "clean" any replies should go into a thread for that post.
+
+Types of content for announcements;
+
+- PRs for Handbook and RFCs
+- Holiday requests
+- WFH requests
+- Sickness
+- Lateness
+- Tea/Coffee
```

Comment:
Why does everybody need visibility on this?

Reply:
I do not need to hear about tea/coffee so I'd kindly ask we don't use Announcements for this. There should be a room for that specifically.